src/components/tracks/track-item.js

```diff
@@ -14,13 +15,17 @@ export default class TrackItem extends Component {
         return "Created";
       case 3:
         return "Deleted";
+      default:
+        //! TODO : RETURN UNDEFINED IS NOT CORRECT
+        return undefined;
```

Comment:
Return `null` instead. There is no a good practice to explicitly assign or return `undefined`